aiida/cmdline/commands/cmd_config.py

```diff
@@ -8,46 +8,230 @@
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
 """`verdi config` command."""
+import textwrap
 
 import click
 
 from aiida.cmdline.commands.cmd_verdi import verdi
 from aiida.cmdline.params import arguments
-from aiida.cmdline.utils import echo
+from aiida.cmdline.utils import decorators, echo
 
 
-@verdi.command('config')
+class DeprecatedCommandGroup(click.Group):
```

Comment:
```suggestion
class DeprecateConfigCommandsGroup(click.Group):
```

I can't be bothered to make it more general lol, because it is just a transient solution for deprecation, that I doubt will be required more generally and will be removed come v2.0